components/fxa-client/src/state_persistence.rs

```diff
@@ -115,6 +116,7 @@ struct V1Config {
     issuer: String,
     jwks_uri: String,
     token_endpoint: String,
+    introspection_endpoint: String,
```

Comment:
Hum this might mean we need to introduce a V2Config, as if JSON doesn't contain `introspection_endpoint` then we'll fail to de-serialize entirely. We could alternativelly make `introspection_endpoint` an `Option<String>` which would avoid the state version bump.

Reply:
The server does not currently advertize this URL; I filed https://github.com/mozilla/fxa/issues/1844 to track doing so.